docs/UserGuide.md

```diff
@@ -81,9 +81,9 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
```

Comment:
Examples provided not changed to match the changed feature.

Reply:
Above examples are under 'add' command.